src/ex6/server/client/src/components/AddTask/item_client.js

```diff
@@ -0,0 +1,32 @@
+export class ItemClient {
+  getItems = async () => {
+    const response = await fetch("http://localhost:3042/items")
+    const todos = await response.json()
+
+    return todos
+  }
+
+  postItem = async (item) => {
+    await fetch("http://localhost:3042/item", {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ item }),
+    })
+  }
+
+  toggleDone = async (item) => {
+    await fetch(`http://localhost:3042/item/${item.id}`, {
```

Comment:
also - export consts whenever possible
the base URL can be in a const as well as the headers
this way you won't have to repeat it